Week2/homework/map_filter.js

```diff
@@ -3,3 +3,14 @@
 const numbers = [1, 2, 3, 4];
 
 // Add your code here
+
+// const newNumbers = [];
+// for (let i = 0; i < numbers.length; i++) {
+//     if (numbers[i] % 2 !== 0) {
+//       newNumbers.push(numbers[i] * 2);
+//     }
+//   }
+//   console.log(newNumbers);
+
+const newNum = numbers.filter(numbers => numbers % 2 !== 0).map(numbers => numbers * 2);
```

Comment:
This is good but I would suggest that next time maybe think of taking another step forward and try to wrap this logic inside a function. For example, having a function(`doubleOddNumbers`) like below would allow you to pass an array of any numbers and another developer who might need a similar behaviour can use your function without having to repeat the logic.
```
const doubleOddNumbers = numbers => numbers
    .filter(numbers => numbers % 2 !== 0)
    .map(numbers => numbers * 2);
```